astroquery/esa/hubble/tests/test_esa_hubble_remote.py

```diff
@@ -103,7 +102,7 @@ def test_hap_composite_to_hap_simple(self):
     def test_hap_simple_to_hap_composite(self):
         esa_hubble = ESAHubble()
         result = esa_hubble.get_member_observations('hst_16316_71_acs_sbc_f150lp_jec071i9')
-        assert result == ['hst_16316_71_acs_sbc_f150lp_jec071']
+        assert result == [' hst_16316_71_acs_sbc_total_jec071', 'hst_16316_71_acs_sbc_f150lp_jec071']
```

Comment:
the space looks superflous

Reply:
indeed the space is necessary, however, that is a big weird behaviour, is it maybe a bug @javier-ballester